src/main/java/uk/org/tombolo/importer/DownloadUtils.java

```diff
@@ -42,13 +46,31 @@ public File fetchFile(URL url, String prefix, String suffix) throws IOException{
 	public InputStream fetchInputStream(URL url, String prefix, String suffix) throws IOException {
 		createCacheDir(prefix);
 		File localDatasourceFile = urlToLocalFile(url, prefix, suffix);
+
+
 		log.info("Fetching local file: {}", localDatasourceFile.getCanonicalPath());
 		if (!localDatasourceFile.exists()){
 			log.info("Local file not found: {} \nDownloading external resource: {}",
 												localDatasourceFile.getCanonicalPath(), url.toString());
-			URLConnection connection = url.openConnection();
-			if (suffix.equals(".json")) connection.setRequestProperty("Accept", "application/json");
-			return new TeeInputStream(connection.getInputStream(), new FileOutputStream(localDatasourceFile));
+
+			URLConnection urlConnection = url.openConnection();
+			if (suffix.equals(".json")) {
+				urlConnection.setRequestProperty("Accept", "application/json");
+			} else {
+				urlConnection.setRequestProperty("User-Agent", "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.11 " +
+								"(KHTML, like Gecko) Chrome/23.0.1271.95 Safari/537.11");
+			}
```

Comment:
doesn't require else block as code in else should run irrespective of '.json'
it can just be
```
if(suffix.equals(".json")){
    urlConnection.setRequestProperty("Accept", "application/json");
}

urlConnection.setRequestProperty("User-Agent", "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.11 (KHTML, like Gecko) Chrome/23.0.1271.95 Safari/537.11");

urlConnection.connect();
```

Reply:
I wish there was an option `accept suggestions` in github 💃